C/common/Types.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _types_H_
+#define _types_H_
+
+typedef unsigned char uint8_t;
+typedef signed   char sint8_t;
+
+typedef enum
+{ 
+	NoError ,
+	FunctioParameterError 
+}Error_t;
+
+#define SET_BIT(_Reg , _BitNum) ( _Reg |=  (1<<_BitNum))
+#define CLR_BIT(_Reg , _BitNum) ( _Reg &= ~(1<<_BitNum))
```

Comment:
I believe theses functions don't relate to `Types.h`. It will be better to add new file called `Utils.h` and add these types of functions